WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginEmailFragment.java

```diff
@@ -448,7 +458,14 @@ public void onSocialChanged(OnSocialChanged event) {
             AppLog.e(T.API, "LoginEmailFragment.onSocialChanged: " + event.error.type + " - " + event.error.message);
             AnalyticsTracker.track(AnalyticsTracker.Stat.LOGIN_FAILED, event.getClass().getSimpleName(),
                     event.error.type.toString(), event.error.message);
-            // TODO: Show error screen.
+
+            switch (event.error.type) {
+                case UNKNOWN_USER:
+                    showErrorDialog(getString(R.string.login_error_email_not_found, mGoogleEmail));
+                    break;
+                default:
```

Comment:
If we're only interested in `UNKNOWN_USER`, let's use a plain `if` statement rather than the `switch`. Otherwise, if there is a reason we catch the `default` let's add a comment about it.

Reply:
Well, I noticed that in the next PR of the cascade this section kinda become more specific so, I'm gonna comment there instead.